subscriptions.go

```diff
@@ -55,6 +55,7 @@ type Subscription struct {
 	PONumber               string              `xml:"po_number,omitempty"`
 	NetTerms               NullInt             `xml:"net_terms,omitempty"`
 	SubscriptionAddOns     []SubscriptionAddOn `xml:"subscription_add_ons>subscription_add_on,omitempty"`
+	PendingSubscription    PendingSubscription `xml:"pending_subscription,omitempty"`
```

Comment:
Could you make this a pointer so the user knows if it's non-nil that it's available?